api/lib/case-list/case-list-helper.ts

```diff
@@ -0,0 +1,39 @@
+/**
```

Comment:
Not sure how much value is added with this extra file? Also the function naming is misleading; we're checking to see if the array contains a value and not where isEmpty. Perhaps move this function to the util.ts file as a generic array helper method and do something like:

```TypeScript
export function isArrayNotEmpty(array) {
   return Array.isArray(array) && array.length && array[0].length > 0
}
```